docs/learn/learn-assets.md

```diff
@@ -58,6 +58,17 @@ transfer is successful.
 
 :::
 
+## Asset Conversion
+
+[An asset conversion pallet](https://paritytech.github.io/polkadot-sdk/master/pallet_asset_conversion/index.html)
+is deployed on AssetHub with Uniswap V2-style functionality. It implements a simple AMM (automated
+market maker), which employs a mathematical formula to determine the price of a token based on its
+ratio within the liquidity pool.
+
+This pallet enables fees to be paid in any asset that has corresponding liquidity (in DOT/KSM) on
+AssetHub, allowing collators to still be paid in native currency. Learn more about asset conversion
```

Comment:
The use cases for asset conversion are broader than just paying collators. Removing it for the time being. There are followup PRs that add more tutorials on Asset Conversion which shall go over the use cases in depth.

```suggestion
AssetHub. Learn more about asset conversion
```